portfolio/src/main/webapp/index.html

```diff
@@ -83,22 +83,15 @@
   </div>
   </div>
   <div class="footer-container">
-    <div class="column1">
-      <a class="footer-item" href="#about-container">About</a>
-      <a class="footer-item" href="#projects-container">Projects</a>
-      <a class="footer-item" href="#work-container">My Work</a>
-      <a class="footer-item" href="#footer-container">Contact</a>
+    <form action="/data" class="contact-form" method="get">
```

Comment:
You'll probably eventually want this to be a post, but since it's not implemented yet, doesn't really matter.

Reply:
yup, on Servlet4 branch I have changed it to a post request